### PR TITLE
#161 Preserve the SRID of allowed area when retrieving it through AccessInfo

### PR DESCRIPTION
**Pull request by taba90**
- Ensure that the SRID is included in the WKT representation of allowedArea, when the AccessInfo object is built from the rule.
- Fix a conflict between JTS versions.

### CODE REVIEW

**Review by etj — CHANGES_REQUESTED** [review]
Are you sure the union will still work ?

https://github.com/geoserver/geofence/blob/master/src/services/core/services-impl/src/main/java/org/geoserver/geofence/services/RuleReaderServiceImpl.java#L210

**Comment by taba90** [comment]
thanks for your review @etj. the necessary logic needed for avoid issue on operation between geometry with different SRID has been added. New test are available and the modification has been lively tested with H2 and PostGIS

File: src/services/core/services-impl/src/test/java/org/geoserver/geofence/services/RuleAdminServiceImplTest.java [hunk 1]
```diff
@@ -28,6 +28,10 @@
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.MultiPolygon;
+import org.locationtech.jts.io.ParseException;
+import org.locationtech.jts.io.WKTReader;
```
Comment (etj) [inline src/services/core/services-impl/src/test/java/org/geoserver/geofence/services/RuleAdminServiceImplTest.java:34]: Added unused imports

**Review by etj — CHANGES_REQUESTED** [review]

File: src/services/core/services-impl/src/test/java/org/geoserver/geofence/services/RuleAdminServiceImplTest.java (hunk 1 above)
Comment (taba90) [reply src/services/core/services-impl/src/test/java/org/geoserver/geofence/services/RuleAdminServiceImplTest.java]: fixed

**Review by etj — CHANGES_REQUESTED** [review]
Also handle REST logic, e.g.:

https://github.com/geoserver/geofence/blob/3.4.x/src/services/modules/rest/impl/src/main/java/org/geoserver/geofence/services/rest/impl/RESTRuleServiceImpl.java#L257-L269

**Comment by taba90** [comment]
fixed

**Pull request merged**